src/umpire/resource/MemoryResourceRegistry.cpp

```diff
@@ -137,27 +137,27 @@ MemoryResourceRegistry::MemoryResourceRegistry() : m_allocator_factories()
     int device_count{0};
     auto error = ::hipGetDeviceCount(&device_count);
     if (error != hipSuccess) {
-      UMPIRE_ERROR("Umpire compiled with HIP support but no GPUs detected!");
-    }
-
-    registerMemoryResource(util::make_unique<resource::HipDeviceResourceFactory>());
-    m_resource_names.push_back("DEVICE");
-
-    for (int device = 0; device < device_count; ++device) {
-      std::string name{"DEVICE::" + std::to_string(device)};
-      m_resource_names.push_back(name);
-    }
+      UMPIRE_LOG(Warning, "Umpire compiled with HIP support but no GPUs detected!");
```

Comment:
Same error code question as above.  Also, how is this being tested?  Has this been manually tested on corona?